src/clojure_by_example/ex05_immutability_and_fp.clj

```diff
@@ -59,26 +56,25 @@ pi ; evaluate to confirm
 
 ;; - So suppose we define...
 
-(def habitable-planets [{:pname "Earth" :moons 1}
```

Comment:
Calling them `habitable-planets` to begin with adds a lot of confusion at line 81, where you evaluate `habitable-planets` and it returns the planets unchanged, which just looks like all planets are habitable.

Reply:
Agreed.